tests.py

```diff
@@ -226,20 +226,27 @@ def test_get_sld_top_convenience_function_is_the_same_as_PublicSuffixList_method
     def test_get_tld_returns_correct_tld_or_etld(self):
         psl = publicsuffix.PublicSuffixList()
         assert 'com' == psl.get_tld('com')
+        assert 'ck' == psl.get_tld('ck')
         assert 'kobe.jp' == psl.get_tld('city.kobe.jp')
-        assert 'kobe.jp' == psl.get_tld('kobe.jp')
-        assert 'amazonaws.com' == psl.get_tld('amazonaws.com')
+        assert 'jp' == psl.get_tld('kobe.jp')
+        assert 'com' == psl.get_tld('amazonaws.com')
```

Comment:
Neither `kobe.jp` nor `amazonaws.com` is in https://publicsuffix.org/list/public_suffix_list.dat .

Reply:
the public suffix list does change regularly, so we have to take some care that we aren't dependent on creating tests that depend on something we don't control. the behavior should be independent of the actual PSL.  Also, as we noted in Vadym's pull request, some of the unit tests on the mozilla site were faulty (didn't follow the stated algorithm).

having said that, *.kobe.jp was in the psl at some point, as are several subdomains of amazon.aws.

Reply:
our tests shouldn't be reliant on moving data. we should use a fixed test file.
otherwise, agree... unless the wildcards are turned off, and then these become entries in the list.

Reply:
> our tests shouldn't be reliant on moving data. we should use a fixed test file.

I perfectly agree with you. However, we should create another issue to solve that problem.